src/PhoneValidation.tsx

```diff
@@ -0,0 +1,73 @@
+import type { ChangeEvent } from 'react'
+import { useRef, useState } from 'react'
+import BackButton from './components/BackButton'
+import ValidationMessage from './components/ValidationMessage/ValidationMessage'
+// import CorrectValidation from './components/ValidationMessage'
+import './PhoneValidation.scss'
+
+type Props = {}
+
+const PhoneValidation = ({}: Props) => {
```

Comment:
```suggestion
const PhoneValidation = () => {
```

No need for props here